plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/frontend.ts

```
@@ -47,12 +50,31 @@ const ALLOWED_ATTR = [
+		| 'sku'
+		| 'weight'
+		| 'dimensions';
 };
 
 const productElementStore = store(
 	'woocommerce/product-elements',
 	{
+		state: {
+			get productData(): ProductData | undefined {
+				if ( ! productDataState?.productId ) {
+					return;
```

Biome (2.1.2)

[error] This return should return a value because it is located in a getter.



(lint/suspicious/useGetterReturn)